crates/threshold-bls-ffi/src/ffi.rs

```diff
@@ -489,6 +490,76 @@ fn serialize<T: Serialize>(in_obj: *const T, out_bytes: *mut *mut u8) -> bool {
     true
 }
 
+/// Generates a t-of-n polynomial and private key shares
+///
+/// The return value should be destroyed with `destroy_keys`.
+///
+/// # Safety
+///
+/// WARNING: This is a helper function for local testing of the library. Do not use
+/// in production, unless you trust the person that generated the keys.
+///
+/// The seed MUST be at least 32 bytes long
+#[no_mangle]
+pub unsafe extern "C" fn threshold_keygen(
```

Comment:
Is the `unsafe` tag on the function still needed?

Reply:
This should be safely removable. Removed.